Perl/Decoder/t/091_thaw_order.t

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env perl
+use 5.020_000;
```

Comment:
Makefile.PL says we're backward compatible back to perl v5.8, so this (and all the `//` and `//=` ops) aren't going to work.

Reply:
ah yeah, good point.

Reply:
Done.